google/provider.go

```diff
@@ -75,7 +76,8 @@ func Provider() terraform.ResourceProvider {
 			"google_compute_region_instance_group": dataSourceGoogleComputeRegionInstanceGroup(),
 			"google_container_cluster":             dataSourceGoogleContainerCluster(),
 			"google_container_engine_versions":     dataSourceGoogleContainerEngineVersions(),
-			"google_active_folder":                 dataSourceGoogleActiveFolder(),
+			"google_gcr_repository":                dataSourceGoogleGcrRepo(),
```

Comment:
should we use the acronym GCR or expand it to `container_registry`.

It is also weird to have google_g(oogle)c(ontainer)r(egistry)_repository. `google` is repeated twice.

Reply:
Agreed - expanded it.